### ISSUE TITLE
Incorrect URL in vignette

### ISSUE THREAD

**Issue by Robinlovelace** [ITSLeeds/pct#82]
Message from CRAN:

```
 Found the following (possibly) invalid URLs:
    URL: http://robinlovelace.net/geocompr (moved to https://geocompr.robinlovelace.net/)
      From: inst/doc/pct_training.html
      Status: 200
      Message: OK
```

**Comment by Robinlovelace** [comment]
Heads-up @natesheehan this should be a quick fix, you up for giving it a go? (Trying to publish the latest version, CRAN is very fussy about URLs.)

**Comment by natesheehan** [comment]
@Robinlovelace Sure will create the PR this afternoon

**Comment by natesheehan** [comment]
https://github.com/ITSLeeds/pct/pull/84

**Comment by Robinlovelace** [comment]
Fixed in #84
